src/public/WRLDBuildingInformation.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+#import <Foundation/Foundation.h>
+
+#import "WRLDBuildingDimensions.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+/*!
+ Information about a building on the map, obtained by adding a WRLDBuildingHighlight object and
+ to the map.
+ */
+@interface WRLDBuildingInformation : NSObject
+
+/*!
+ A unique identifier for the building. The BuildingId for a building on the map is not
+ necessarily maintained between versions of the map or Api.
+
+ @returns A unique identifier for the building.
+ */
+- (NSString*) buildingId;
```

Comment:
Could these be properties? Similar to https://github.com/wrld3d/ios-api/blob/master/src/public/WRLDPolygon.h#L84